demo/src/App.tsx

```diff
@@ -25,8 +25,11 @@ export default function App() {
 
   useEffect(() => {
     //Workaround for issue with non-working NativeEventEmitter if created from library code
-    const nativeEventEmitter = new NativeEventEmitter(DIDCommResolversProxyModule)
-    DIDCommResolversProxy.start(nativeEventEmitter)
+    const emitter =
```

Comment:
I still have the feeling that we can avoid using condition here. The problem with using `DIDCommResolversProxyModule` in iOS may be related to logic where every resolver proxy creates it's own instance of this module.